app/controllers/user_games_controller.rb

```diff
@@ -3,14 +3,22 @@ class UserGamesController < ApplicationController
   def create
     game = Game.find(params[:format])
     if !current_user 
-      flash[:notice] = "You must sign in to add games to Your Games"
+      flash[:notice] = "You must sign in to add games to Saved Games"
     elsif UserGame.user_game_exists?(game.id, current_user.id)
-      flash[:notice] = "You have already added #{game.title} to Your Games"
+      flash[:notice] = "You have already added #{game.title} to Saved Games"
     else #UserGame.find_by(game_id: param([:id]), user: current_user)
       user_game = UserGame.create(game_id: game.id, user_id: current_user.id)
-      flash[:notice] = "You have added #{game.title} to Your Games"
+      flash[:notice] = "You have added #{game.title} to Saved Games"
     end 
     redirect_back(fallback_location: games_path)
   end
 
+  def destroy 
+    game = Game.find(params[:id])
+    user_game = UserGame.find_by(user_id: current_user.id, game_id: game.id)
```

Comment:
I believe we aren't supposed to have anything but `find` in a controller. I suppose this could get moved to a model. but `done > perfect`